beetsplug/mosaic.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2015-2016, Ohm Patel.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+"""Allows beets to create a mosaic from covers."""
+from __future__ import division, absolute_import, print_function
+
+import os.path
+
+from beets.plugins import BeetsPlugin
+from beets import ui
+
+from PIL import Image
+import math
+from parse import *
```

Comment:
It's usually best to avoid "star imports," which make it difficult to tell where names come from.